tsconfig.base.json

```diff
@@ -228,7 +228,8 @@
       "@energinet-datahub/gf/util-browser": [
         "libs/gf/util-browser/src/index.ts"
       ],
-      "@energinet-datahub/watt": ["libs/ui-watt/src/index.ts"]
+      "@energinet-datahub/watt": ["libs/ui-watt/src/index.ts"],
+      "rxjs": ["./node_modules/rxjs"]
```

Comment:
Question: why is this necessary?

Reply:
RxJSv6 is the default version of RxJS in MSAL-Angular v2. In order to use the installed version of RxJSv7 for this sample, "rxjs": ["./node_modules/rxjs"] must be added to the compilerOptions object's paths object in tsconfig.json. This tells the compiler to use the version of RxJS specified in package.json (RxJSv7).

Reply:
Exactly. I hope it works :)